tests/target-browser/target_browser_test.go

```diff
@@ -272,13 +273,162 @@ var _ = Describe("Target Browser Tests", func() {
 			It(fmt.Sprintf("Should fail if flag %s is provided but %s is not provided", cmd.CertificateAuthorityFlag, cmd.UseHTTPS), func() {
 				isLast = true
 				args := []string{cmdGet, cmdBackupPlan, flagCaCert, filepath.Join(testDataDirRelPath, tlsCertFile)}
-				args = append(args, commonArgs...)
+				args = append(args, commonArgsWithJSONOutputFormat...)
 				command := exec.Command(targetBrowserBinaryFilePath, args...)
 				output, err := command.CombinedOutput()
 				Expect(err).Should(HaveOccurred())
 				Expect(string(output)).Should(ContainSubstring("Client.Timeout exceeded while awaiting headers"))
 			})
 		})
+
+		Context(fmt.Sprintf("Backup & BackupPlan test-cases for %s flag", cmd.OutputFormatFlag), func() {
+			var (
+				backupPlanUIDs                   []string
+				noOfBackupPlansToCreate          = 1
+				noOfBackupsToCreatePerBackupPlan = 1
+				once                             sync.Once
+				isLast                           bool
+				tvkInstanceUID                   string
+				backupPlanUID                    = "f7f17b64-320f-43f8-8ca3-234f157df9d5"
+				defaultNoOfColumns               = 5
+				backupRow                        = []string{"sample-backup", "Backup", "529efb26-9f06-443a-a263-c42f1af04493",
+					"Full", "Available", backupPlanUID, tvkInstanceUID, "2021-05-10T20:10:40Z", "2021-05-10T20:11:06Z"}
+				backupPlanRow = []string{"bp11", "BackupPlan", backupPlanUID, "Custom", tvkInstanceUID,
+					"2021-05-10 20:11:06 +0000 UTC", "2021-05-10T20:03:55Z"}
+			)
+			BeforeEach(func() {
+				once.Do(func() {
+					createTarget(true)
+					tvkInstanceUID = guid.New().String()
+					createBackups(noOfBackupPlansToCreate, noOfBackupsToCreatePerBackupPlan, tvkInstanceUID, "mutate-tvk-id")
+					backupPlanUIDs = verifyBackupPlansAndBackupsOnNFS(noOfBackupPlansToCreate, noOfBackupPlansToCreate*noOfBackupsToCreatePerBackupPlan)
+					verifyBrowserCacheBPlan(noOfBackupPlansToCreate)
+				})
+			})
+
+			AfterEach(func() {
+				if isLast {
+					deleteTarget(false)
+					removeBackupPlanDir(backupPlanUIDs)
+				}
+			})
+			It(fmt.Sprintf("Should fail cmd BackupPlan if flag %s is given without value", cmd.OutputFormatFlag), func() {
+				args := []string{cmdGet, cmdBackupPlan}
+				args = append(args, commonArgs...)
+				args = append(args, flagOutputFormat)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagOutputFormat)))
+			})
+
+			It(fmt.Sprintf("Should fail cmd backupPlan if flag %s is given invalid value", cmd.OutputFormatFlag), func() {
+				args := []string{cmdGet, cmdBackupPlan, flagOutputFormat, "invalid"}
+				args = append(args, commonArgs...)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
```

Comment:
Instead of this use funcs `runCmdBackupPlan`, etc there might be throttling msg logs in output and test will fail while comparing output.. do the same at other places